### ISSUE TITLE
[FEATURE] - Add `inline` commands to delete/update projects with invalid path

### ISSUE THREAD

**Issue by ctf0** [alefragnani/vscode-project-manager#569]
right now we have to clear the projects manually through the ext json file

it would be much better & quicker if the ext check for the projects paths on vscode launching & remove the unfound ones.

another solution could be adding an `x` icon next to each project so we can delete any one we dont need anymore.

**Comment by alefragnani** [comment]
Hi @ctf0 ,

It already have a _check_, but it is only available when you use the `Project Manager: List Projects to Open` and `Project Manager: List Projects to Open in New Window` commands, via the `Command Palette`. It adds a **"Path does not exist"** message below the project name/path. 

You can however, turn off this validation, using the `projectManager.checkInvalidPathsBeforeListing` setting as `false`.

This _check_ however is not being used in the Side Bar, which I guess is the reason you opened this issue, right?

**Comment by ctf0** [comment]
yep, but also the check from the command panel still needs the manual way, however i was hoping that the check and removal would happen automatically on the extension start.

**Comment by alefragnani** [comment]
Instead of the extension taking care of the so called missing projects, and wipe them out from the project list, I prefer a _notification_ approach. Leave to the user the decision about what to do with it, delete or update the path. 

That's what happens today when you select a project with a **Path does not exist** message.

To make the clean up easier (individually), I guess a few thing could be added to _missing projects_.

1. Improve the Side Bar
  * Replace the icon, using the same one displayed in the Command Palette
  * Add a "path does not exist" message in the Project name
  * Add `Delete` and `Update Path` inline commands for these projects

2. Improve the Command Palette
  * Add the same `Delete` and `Update Path` as inline commands, using the newly released API (#570)

Alternatively, a new `Delete projects with invalid path` command, only in the Side Bar. This command, however, would only work for _local_ projects. Any remote project (Container, SSH, WSL and Codespaces) would be ignored.

**Comment by ctf0** [comment]
> Add Delete and Update Path inline commands for these projects

would be enough for the sidebar

**Comment by ctf0** [comment]
any updates ?
